jerry-debugger/src/lib/cdt-proxy.ts

```diff
@@ -16,13 +16,21 @@ import * as WebSocket from 'ws';
 import WebSocketServer = WebSocket.Server;
 import * as rpc from 'noice-json-rpc';
 import Crdp from 'chrome-remote-debug-protocol';
+import * as http from 'http';
+import uuid from 'uuid/v1';
 
 export interface ChromeDevToolsProxyServerOptions extends WebSocket.ServerOptions {
+  jsfile: string
 }
 
+// TODO: where to put this?
+var SERVER_PORT = 9229;
+var SERVER_UUID = uuid();
+
 export class ChromeDevToolsProxyServer {
   constructor(options: ChromeDevToolsProxyServerOptions) {
-    const wss = new WebSocketServer(options);
+    const server = http.createServer();
+    const wss = new WebSocketServer({ server });
```

Comment:
I'd change this to something along these lines:
```
const sanitizedOptions = {
  port: options.port || DEFAULT_SERVER_PORT,
  uuid: options.uuid || uuid(),
  jsfile: options.jsfile || 'untitled.js',
};
const wssOptions = {
  port: sanitizedOptions.port,
  uuid: sanitizedOptions.uuid,
  server,
};
const wss = new WebSocketServer(wssOptions);
```

I think it makes sense to generate a new uuid for each instance.

Reply:
I may have done it inadvertently but apparently passing only { server } to WSS was exactly what needs to happen. If you pass port, it tries to listen on that itself and I can no longer use both HTTP and WSS server. But if you pass just server it piggybacks on that.